tools/rust_api/Cargo.toml

```diff
@@ -2,47 +2,47 @@
 name = "kuzu"
 version = "0.9.0"
 description = "An in-process property graph database management system built for query speed and scalability"
-# Note: 1.72 required for testing due to latest dependencies of the arrow feature
-rust-version = "1.67"
+keywords = ["database", "graph", "ffi"]
 readme = "kuzu-src/README.md"
-homepage = "http://kuzudb.com/"
+homepage = "https://kuzudb.com/"
 repository = "https://github.com/kuzudb/kuzu"
 license = "MIT"
 categories = ["database"]
+
+rust-version = "1.81"
+edition = "2021"
+
 # Only include files required to build to keep crate size small
 include = [
     "build.rs",
     "/src",
     "/include",
-# If more files are added to kuzu-src, they should also be added to the rerun-if-changed
-# instructions in build.rs so that rebuilds are detected properly
+    # If more files are added to kuzu-src, they should also be added to the rerun-if-changed
+    # instructions in build.rs so that rebuilds are detected properly
     "/kuzu-src/src",
     "/kuzu-src/cmake",
     "/kuzu-src/third_party",
     "/kuzu-src/CMakeLists.txt",
     "/kuzu-src/tools/CMakeLists.txt",
 ]
 
-edition = "2018"
-
 [dependencies]
-cxx = "1.0"
+arrow = { version = "55", optional = true, default-features = false, features = ["ffi"] }
+cxx = "=1.0.138" # latest version that builds on clang 15
+rust_decimal = { version = "1.37", default-features = false }
```

Comment:
Why bump the rust_decimal version? We haven't changed anything we do with rust_decimal since adding it, so it's unlikely that it's no longer working with 1.35, and this keeps things compatible with another crate if they need to use 1.35/1.36 for some particular reason (E.g. the above cxx pin). We could probably even push the requirement back earlier since [the only function we use was introduced in version 1.2.1](https://docs.rs/rust_decimal/1.2.1/rust_decimal/struct.Decimal.html#method.from_i128_with_scale) (I haven't tested it with that version though).

Reply:
I was following the "best practice" of updating frequently so that the ecosystem is not stuck on older version, esp through transitive dependencies.

Reply:
This doesn't force 1.35 though; the default is the `^` operator which would mean `>=1.35,<2`